Infrastructure/DTOs/PlayerDTO.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Infrastructure.DTOs;
+
+public class PlayerDTO
+{
+    public int? Id { get; set; }
+    public string Name { get; set; } = string.Empty;
```

Comment:
Why to have it default empty, but not null?

Reply:
because of error: Cannot convert null literal to non-nullable reference type

Reply:
It's not an error, but compiler warning. Type `string` is a reference type, it can be null, just to suppress a warting, use `!` operator: `public string Name { get; set; } = null!;` or `public string Name { get; set; } = default!;`

Reply:
fixed